client-data/js/board.js

```diff
@@ -310,6 +311,32 @@ function removeWhiteboardSnapshot() {
 	if (elem) Tools.drawingArea.removeChild(elem);
 }
 
+/**
+ * update background to apple whiteboard image to avoid disappontment
+ * between white prestine background and yellowish background of the whiteboard.
+ */
+function updateWhiteboardBackground() {
```

Comment:
It's probably better to put this in robottools.js since we want to call it from other files.